### DISCUSSION TITLE
taking in the document structure into the chunks - possible?

### DISCUSSION THREAD

**Discussion by mgrabmayr** [llmware-ai/llmware#190] [Q&A]
Hi
I love the thought of having image, text and tables linked in a structure to also then use proximity as a measure on retrieval. I was thinking to also bring in the hierarchy of documents (via headings) into the chunks and nodes. I hope that his can get increase the clarity on retrieval.
Is this something that can be somewhat simply realised with llmware?
and i was planning on postgres as a DB - do you have this on your roadmap with an ETA?

**Comment by NYDocutest** [comment]
Hi Thank you for your question. We are in the process  evaluating Postgres as our next DB to support so your question is pretty timely! We are working hard to get some really exciting features out shortly then plan to focus on Postgres hopefully soon. We will update you when we have more details.

**Comment by NYDocutest** [comment]
Sorry for the late update but we have started supporting Postgres a couple of weeks ago!
